app/src/main/java/com/trainstation/frettirtest2/MyViewPagerFragment.java

```diff
@@ -50,8 +42,10 @@ public void onCompleted() {
 
             @Override
             public void onError(Throwable e) {
-                // TODO: show toast
-                Log.e("MyViewPagerFragment", "Couldn't get response", e);
+                Context context = getContext();
```

Comment:
when you have a fragment you can always call this `getContext()` method, it is a method that is declared in the Android Fragment (remember that we extend Fragment: https://github.com/helgio/frettaApp/pull/2/files#diff-bd250610c6b3fa979c8e0f1b71926f40R23 so we can always use methods that is declared there)

**Remember also that you can get the context from an activity**, i.e. if you have an activity you can use that as a context, and to get an activity
- you can use `this` if you are inside activity
- you can use `getActivty` if you are inside fragment (that method will return the activity that the fragment is attached to)